Sources/NIOPerformanceTester/main.swift

```diff
@@ -19,6 +19,9 @@ import NIOFoundationCompat
 import Dispatch
 import NIOWebSocket
 
+// Use unbuffered stdout to help detect exactly which test was running in the event of a crash.
+setbuf(stdout, nil)
+
```

Comment:
Not sure this is relevant in this case, but macOS outputs test results to stderr

Reply:
Hmm. This isn’t an XCTest thing; this is just a Swift binary with a main.swift using `print`. Are you sure that goes to stderr on macOS?

Reply:
Ah sorry. Should have realised that.

Reply:
That’s good to know for future, thanks!